build/tools/esp32/esptool_py/Makefile

```diff
@@ -130,6 +130,10 @@ erase_flash: $(ESPTOOLPY_SUBMODULE)
 	@echo "Erasing entire flash..."
 	$(ESPTOOLPY_SERIAL) erase_flash
 
+erase_region: $(ESPTOOLPY_SUBMODULE)
```

Comment:
1. Could you add the way of using this command in guidance?
2. Is this working with partition name?

Reply:
@ChinaTizenRT could you check above?

Reply:
okay, i will improve it,

Reply:
Could you add the way of using this command in guidance?
----if no any argument, then show guide

Is this working with partition name?
-- yes, it now supports partion index.